packages/orchestration/src/orchestration-api.ts

```diff
@@ -146,6 +146,7 @@ export interface Orchestrator {
     IssuingChain extends keyof KnownChains,
   >(
     denom: Denom,
+    holdingChainName: HoldingChain,
```

Comment:
+cc @dtribble @mitdralla

Reply:
why is `holdingChainName` constrained to a static list of known chains?

OK, I guess I see why for now. Tolerable tech debt, I suppose.

In due course, it seems like `getDenomInfo` should be parameterized by...

```
  getDenomInfo: <
    HoldingChainInfo extends ChainInfo,
    IssuingChainInfo extends ChainInfo,
  >(
```

and maybe even the `ConnectionInfo` between them.

Reply:
> why is holdingChainName constrained to a static list of known chains?

Something to potentially solve in #9992. In the (stale) PR that attempted to close it, allowing `string` for `chainName` was an approach discussed.